### ISSUE TITLE
Refresh airspace

### ISSUE THREAD

**Issue by roeles** [skylines-project/skylines#1408]
I recently updated the Dutch airspace over at xcsoar-data repositories.
Can the tiles for Skylines be re-generated, or should they already be up-to-date?

**Comment by Turbo87** [comment]
yes, they will need to be updated. skylines does unfortunately not use the `xcsoar-data` repo (yet).

**Comment by roeles** [comment]
Okay. Airspace is downloadable at:
http://download.xcsoar.org/airspaces/EH1904.txt

**Comment by Turbo87** [comment]
```
* Copyright: no modification, distribution or publication #
* of this file is allowed without explicit prior approval #
* of the maker of this file.				  #
```
are we allowed to put that file on the XCSoar server?

**Comment by roeles** [comment]
Yes, I obtained written permission a couple of years ago. I also inform the
author every year of my efforts (and results) to update XCSoar.

**Comment by roeles** [comment]
Updated:
http://download.xcsoar.org/airspaces/EH1904A.txt

**Comment by lordfolken** [comment]
### Data upstream note (2026-09-18)

Preferred source after Platform is green: [XCSoar/xcsoar-data-content](https://github.com/XCSoar/xcsoar-data-content) (waypoints + airspace; feeds download.xcsoar.org). `xcsoar-data-repository` is deprecated.

Related: #2152, #2483, #2488. No importer work until CI/Docker are green — then prefer a thin import from that repo/artifacts over a parallel Welt2000-only path.

**Comment by lordfolken** [comment]
### Data currency cluster (linked)

Hub for airfields/airspace refresh after Platform. Prefer [XCSoar/xcsoar-data-content](https://github.com/XCSoar/xcsoar-data-content).

**Related issues**
- #2152 — airspace list inconsistent with map
- #2483 — Japan airspace/airfield update (regional; import fallout)
- #2488 — add airport (Invermere; import fallout)

One-off airport/region asks fold into this refresh path rather than separate importers.
